CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix container not starting because Superset was starting too slow and was killed because a failing liveness probe.
+We now add a proper startup probe, which allows Superset to take longer to start up ([#654]).
```

Comment:
Indenation

```suggestion
- Fix container not starting because Superset was starting too slow and was killed because a failing liveness probe.
  We now add a proper startup probe, which allows Superset to take longer to start up ([#654]).
```

But I think it could be briefer:

```suggestion
- Add startup probe to give Superset longer to start up than the default ([#654]).
```

Reply:
I prefer the current two sentences as they make a bit clearer what the problem was exactly and what the impact was.
> than the default

There is no default, only before (like 60s) and after (~10 minutes) ^^

Reply:
> There is no default, only before (like 60s) and after (~10 minutes) ^^

Where is the 60 defined?

Reply:
15s initial_delay_seconds + failure_threshold (3) * period_seconds (15s)

Reply:
I don't see where those are defined :sweat_smile:

I feel like there should be a comment explaining how to derive the values for the next person who comes along (even better if it can be done using `consts` or some runtime vars.

Reply:
Created https://github.com/stackabletech/operator-rs/pull/1078